### PR TITLE
[Documentation] Examiner Test Plan updated

### LINKED ISSUE #6603: [Examiner] Add detail to Test Plan for examiner module
Some detail can be added to the examiner module to include specific cases.

### PR DESCRIPTION
**Pull request by AlexandraLivadas**
Some steps were added to the examiner module test plan for clarity.

* Resolves #6603

### CODE REVIEW

**Comment by AlexandraLivadas** [comment]
Per discussion with @spell00, some more steps will be added to cover editing an existing examiner.

File: modules/examiner/test/TestPlan.md [hunk 1]
```diff
@@ -8,4 +8,15 @@
 4. Check "Clear Filters" button works well.
    [Automation Testing]
 5. Test Add a new examiner.
-   [Automation Testing]
+   [Automation Testing]  
+    a. Check that if you try to add an examiner with no name or no site, a warning message appears.
```
Comment (christinerogers) [inline modules/examiner/test/TestPlan.md:12]: Is Project a filter option yet in this module?  If so, add tests for these permissions.

File: modules/examiner/test/TestPlan.md [hunk 2]
```diff
@@ -8,4 +8,15 @@
...
+    b. A success message should appear when the examiner is created.  
+    c. Check that the new examiner appears on the examiner list. 
+6. Test Edit Examiner.  
+    a. Click on an existing examiner's name and check that it takes you to the "Edit Examiner" page.    
+    b. Fill out the BMI and Radiology Review sections, as well as the dates, and press "Save". There
+    should be no error.  
+    c. Press the "Reset" button and check that the information you filled out in step B appears in the Change Log.    
+    d. Try to change either the BMI or Radiology Review back to the "N/A" option and check that an error message appears.     
+    e. Change the status of either BMI or Radiology Review to "Certified" and return to the examiner list page. Check
+    that the examiner now has that certification listed.
```
Comment (christinerogers) [inline modules/examiner/test/TestPlan.md:22]: I think our convention is to put `[Manual Testing]` after points like both 5 and 6point but not sure  - it should be clear from other test plans.

File: modules/examiner/test/TestPlan.md (hunk 1 above)
Comment (AlexandraLivadas) [reply modules/examiner/test/TestPlan.md:12]: No, Project is not a filter option.

**Comment by christinerogers** [comment]
@spell00 please review and if you approve, please also tag it as `Passed Manual Tests`

**Pull request merged**